add_tokens.py

```diff
@@ -0,0 +1,22 @@
+import pandas as pd
+
+def load_data():
+    try:
+        data = pd.read_csv('additiona_tokens.csv',index_col=[0])
+    except:
```

Comment:
It's going to be just never okay to use a bare `except:`, and even less okay to return an empty dataframe in this case. This function should be a constant. Not entirely sure, but something like:

```
ADDITIONAL_TOKENS = {
    "uniswap_v2": "pepe",
    "uniswap_v3": "pepe",
}
```

(it should look like the above because it will be formatted with `black`)

Reply:
Changed to printing out Exception details and replaced empty df with a constant